lib/services/game_service.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:niira/models/game.dart';
+
+class GameService {
+  Game currentGame;
+  GameService({this.currentGame});
+
+  void setCurrentGame(Game _selectedGame) => currentGame = _selectedGame;
```

Comment:
its a bit more boiler plate, but i really like having this method to call when setting the current game

Reply:
bit more (one line lol)